manifests/config.pp

```diff
@@ -31,4 +31,17 @@
     group  => 'root',
     mode   => '0755',
   }
+
+  foreman_config_entry { 'pulp_client_cert':
+    value          => $client_cert,
+    ignore_missing => false,
+    before         => Exec['foreman-rake-db:migrate'],
```

Comment:
We will want this happen afterward, since this performs a change in the database, we want a migrated database. I might even put this after `db:seed` just to be safe.